File: mybatis-plus-join-test/test-join/src/main/java/com/github/yulichang/test/join/mapper/MyBaseMapper.java
package com.github.yulichang.test.join.mapper;

import com.github.yulichang.base.MPJBaseMapper;
import org.apache.ibatis.annotations.Mapper;

import java.util.List;

@Mapper
public interface MyBaseMapper<T> extends MPJBaseMapper<T> {

    @SuppressWarnings("UnusedReturnValue")
    int insertBatchSomeColumn(List<T> entityList);
}

File: mybatis-plus-join-test/test-kotlin/src/main/kotlin/com/github/yulichang/test/kt/mapper/MyBaseMapper.kt
package com.github.yulichang.test.kt.mapper

import com.github.yulichang.base.MPJBaseMapper
import org.apache.ibatis.annotations.Mapper

@Mapper
interface MyBaseMapper<T> : MPJBaseMapper<T> {
    fun insertBatchSomeColumn(entityList: List<T>?): Int
}
